src/main/java/core/basesyntax/service/impl/ReadImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadImpl implements Reader {
+    @Override
+    public List<String> fileReader(String file) {
+        List<String> dataShop;

```

Comment:
```suggestion
        List<String> fiuleData;

```

we can use this method not only for shop